src/node_options.h

```diff
@@ -138,6 +138,7 @@ class EnvironmentOptions : public Options {
   bool heap_prof = false;
 #endif  // HAVE_INSPECTOR
   std::string redirect_warnings;
+  std::string output_directory;
```

Comment:
`diagnostic_dir`, cf. 

```
% grep _dir src/node_options.cc 
    if (!cpu_prof_dir.empty()) {
    if (!heap_prof_dir.empty()) {
            &EnvironmentOptions::cpu_prof_dir);
            &EnvironmentOptions::heap_prof_dir);
            &PerProcessOptions::report_directory,
            &PerProcessOptions::icu_data_dir,
```

As you can see, I rely heavily on grep for quick consistency checks! :-)